package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "pushwoosh-cordova-plugin",
-  "version": "7.13.0-OS1",
+  "version": "7.17.1",
```

Comment:
Should keep the -OS suffix

Reply:
@EiyuuZack, @usernuno is this just an upgrade from upstream without changes on our end? because if it is, and if I recall correctly, then the suffix isn't needed. The suffix is added only when our branch/repo deviates from upstream.

Reply:
It contains our changes to plugin.xml to lock Android support libs to API 26